libraries/app/include/graphene/app/api.hpp

```diff
@@ -325,12 +393,22 @@ namespace graphene { namespace app {
                                              uint8_t min_bits,
                                              uint64_t actual_value );
                                        
-         
+         /**
+          * @brief Verifies range proof rewind for 33-byte commit
+          * @param nonce Sha-256 blind refactor type
+          * @param commit 33-byte commitment
+          * @param proof List of characters
+          * @return A structure with success, min, max, value_out, blind_out and message_out values
+          */
          verify_range_proof_rewind_result verify_range_proof_rewind( const blind_factor_type& nonce,
                                                                      const fc::ecc::commitment_type& commit, 
                                                                      const std::vector<char>& proof );
          
-                                         
+         /**
+          * @brief Get range proof info
+          * @param proof List of characters
+          * @return A range proof info structure with exponent, mantissa, min and max values
+          */                                       
```

Comment:
Do you understand how the crypto_api works?
Your documentation provides nothing useful that isn't obvious from the method/variable names.

Reply:
not all details still researching this code in more details ... blind transactions are main idea for `crypto_api`